pkg/controller/plan/kubevirt.go

```diff
@@ -712,6 +741,13 @@ func (r *KubeVirt) virtualMachine(vm *plan.VMStatus) (object *cnv.VirtualMachine
 			vm.String())
 		object = r.emptyVm(vm)
 	}
+	//Add the original name and ID info to the VM annotations
+	if len(originalName) > 0 {
+		annotations := make(map[string]string)
+		annotations[AnnOriginalName] = originalName
+		annotations[AnnOriginalID] = vm.ID
+		object.ObjectMeta.Annotations = annotations
```

Comment:
currently added to all VMS as a label, I've added also an annotation to make it easier for users to see both fields at the same place, but not really needed.